src/main/java/cn/nukkit/item/ItemBanner.java

```diff
@@ -76,6 +76,10 @@ public int getPatternsSize() {
         return (this.hasCompoundTag() ? this.getNamedTag() : new CompoundTag()).getList("Patterns").size();
     }
 
+    public boolean hasPattern() {
```

Comment:
Please add `PowerNukkitOnly` and `Since` attributes

Reply:
Okay